kolibri/content/serializers.py

```diff
@@ -130,7 +130,9 @@ def get_next_content(self, target_node):
         return {'kind': root.kind, 'id': root.id}
 
     def get_license_description(self, target_node):
-        return target_node.license.license_description
+        if target_node.license:
+            return target_node.license.license_description
+        return ''
```

Comment:
Might be better to return `None` here, as empty strings are not falsy in JSON/Javascript.